simulation/README.md

```diff
@@ -7,15 +7,6 @@ Additionally, it publishes a [TwistStamped](http://docs.ros.org/melodic/api/geom
 ### Assumptions
 This readme assumes that you are able to mount the git repo in the docker container, and that you have the most recent Docker image pulled. 
```

Comment:
Change to assuming the containers are running for [local testing](https://github.com/DukeRobotics/robosub-ros#local-testing) as specified in the main README. The current assumptions should no longer need to be stated as long as they've followed the instructions in the main readme.